backend/dal/Helpers/Extensions/ProjectExtensions.cs

```diff
@@ -343,7 +343,7 @@ public static bool IsProjectEditable(this Entity.Project project, ClaimsPrincipa
         /// <returns></returns>
         public static bool IsProjectInDraft(this Entity.Project project, ProjectOptions options)
         {
-            return options.DraftWorkflows.Contains(project.Workflow.Code);
+            return options.DraftWorkflows.Contains(project.Workflow?.Code);
```

Comment:
Should this be changed? It was throwing an error when deleting properties for being NULL - but wondering if there might be a bigger underlying issue

Reply:
If `Workflow == null` it indicates a problem with the loading of the data for this specific use-case.  Essentially the code hasn't made a request to the DB for the `project.Worfklow`.  Which is a problem.

You should update with the following.
```
if (project.Workflow == null) throw new ArgumentNullException(nameof(project), "The 'Workflow' cannot be null.");
return options.DrafWorkflows.Contains(project.Workflow.Code);
```

Reply:
Gotchya, thank you!